tools/flake8.lock

```diff
@@ -180,18 +180,18 @@
             "wheel; extra == \"testing-integration\""
           ],
           "requires_python": ">=3.7",
-          "version": "62.3.2"
+          "version": "62.5"
         }
       ],
       "platform_tag": [
         "cp310",
         "cp310",
-        "manylinux_2_31_aarch64"
+        "macosx_12_0_x86_64"
```

Comment:
FYI: No need to fix but this platform-tag alternating depending on where the lock file is generated is a currently known issue in pex and pants (https://pantsbuild.slack.com/archives/C046T6T9U/p1654207727719299, https://github.com/pantsbuild/pex/issues/1685)